optashift-employee-rostering-docs/src/main/asciidoc/Introduction/WhatIsOptaShift/WhatIsOptaShift-section.adoc

```diff
@@ -3,7 +3,7 @@
 :imagesdir: ../..
 
 Every organization faces planning problems: providing products or services with a
-limited set of _constrained_ resources (employees, assets, time and money). One such
-planning problem is employee shift rostering: assigning shifts to employees who are able
-to perform the shift. OptaShift is a web application that solves employee shift rostering
-problems using https://www.optaplanner.org[OptaPlanner].
+limited set of _constrained_ resources (employees, assets, time and money).
+One such planning problem is employee shift rostering: assigning shifts to employees.
+OptaShift is a web application and REST service that solves employee shift rostering
+problems using the https://www.optaplanner.org[OptaPlanner engine].
```

Comment:
@Christopher-Chianelli It's using https everywhere as far as I can tell. We should indeed use https (never http).

Reply:
Then change the png/svg to reflect that